open-apparel-registry/open-apparel-registry
# Enable users to update approved claimed facility profiles & display claim info in the details sidebar

## Overview

This PR enables users with approved facility claims to edit these claims using a form on the /claimed/claimID route and also updates the facility details sidebar to display that info beneath the existing facility data.

I spun off the work of sending emails to a separate card.

I also spun off adding some multiselect tag fields to a separate card, since we don't really yet know the set of tags for these fields. It occurred to me that we can also remove & defer adding the `minimum_order_quantity` and `average_lead_time` fields, too, for now since users can currently just include this info in a free form way in the `facility_description` field.

Connects #518 

## Demo

![Screen Shot 2019-06-12 at 1 59 23 PM](https://user-images.githubusercontent.com/4165523/59374689-6e420f80-8d1a-11e9-9937-80a7a5f49306.png)


![Screen Shot 2019-06-11 at 9 13 22 PM](https://user-images.githubusercontent.com/4165523/59316728-e44c6500-8c8d-11e9-83fc-088df7634660.png)

## Testing Instructions

- get this branch, then `resetdb`, `processfixtures`, switch on the claim a facility and create a superuser
- sign up for an account through the UI, then fill out a claim a facility form
- sign in as the superuser and approve the claim
- sign back in as the other user and visit the OAR ID for the facility you have claimed. Clicking on the claim badge will now take you to your approved claim (if signed out, clicking on the badge will go to a not-yet-existing `/about/claimedfacilities` route
- verify that the claimed facility profile form loads and that you can edit and save the fields and that those changes get reflected accurately in the facility details UI

## Checklist
